Updating ASR public docs for H2A and UAMI scenarios. (#127411)
Azure/azure-content: articles/site-recovery/azure-to-azure-support-matrix.md

Premium storage | Supported | Use Premium Block Blob storage accounts to get High Churn support. For more information, see [Azure VM Disaster Recovery - High Churn Support](./concepts-azure-to-azure-high-churn-support.md).
Region |  Same region as virtual machine  | Cache storage account should be in the same region as the virtual machine being protected.
Subscription  | Can be different from source virtual machines | Cache storage account must be in the same subscription as the source virtual machine(s). <br> To use cache storage from the target subscription, use PowerShell.

> Azure Storage firewalls for virtual networks  | Supported | If you're using firewall enabled cache storage account or target storage account, ensure you ['Allow trusted Microsoft services'](../storage/common/storage-network-security.md#exceptions).<br></br>Also, ensure that you allow access to at least one subnet of source Vnet.<br></br>Note: If you are using User Assigned Managed Identity (UAMI) created on recovery services vault, do not restrict virtual network access to your storage accounts used for Site Recovery. You should allow access from 'All networks' if you use vault UAMI.

Reviewer: ```suggestion
Azure Storage firewalls for virtual networks  | Supported | If you're using firewall enabled cache storage account or target storage account, ensure you ['Allow trusted Microsoft services'](../storage/common/storage-network-security.md#exceptions).<br></br>Also, ensure that you allow access to at least one subnet of source Vnet.<br></br>Note: If you're using User Assigned Managed Identity (UAMI) created on recovery services vault, don't restrict virtual network access to your storage accounts used for Site Recovery. You should allow access from 'All networks' if you use vault UAMI.
```
